src/Components/SmartComponents/CVEs/CVEsTableToolbar.test.js

```diff
@@ -23,6 +23,11 @@ jest.mock('../../../Store/Actions/Actions', () => ({
     }) }
 ));
 
+jest.mock('../../../Helpers/Hooks', () => ({
+    ...jest.requireActual('../../../Helpers/Hooks'),
+    useFeatureFlag: () => true 
```

Comment:
It would also make sense to add tests with the feature set to false. 

We might also want to investigate a wrapper function that we can use to run tests with both the feature turned on and off without having to duplicate the tests.